src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -426,8 +426,8 @@ Captions"</string>
     <string name="revanced_enable_wide_search_bar_summary_on">Wide search bar is enabled.</string>
     <string name="revanced_enable_wide_search_bar_summary_off">Wide search bar is disabled.</string>
     <string name="revanced_enable_wide_search_bar_with_header_title">Enable wide search bar with header</string>
-    <string name="revanced_enable_wide_search_bar_with_header_summary_on">Wide search bar includes YouTube header.</string>
-    <string name="revanced_enable_wide_search_bar_with_header_summary_off">Wide search bar does not include YouTube header.</string>
+    <string name="revanced_enable_wide_search_bar_with_header_summary_on">Wide search bar does not hide the YouTube header.</string>
+    <string name="revanced_enable_wide_search_bar_with_header_summary_off">Wide search bar hides the YouTube header.</string>
```

Comment:
Alternatively, it can be like this or something similar:

```suggestion
    <string name="revanced_enable_wide_search_bar_with_header_summary_on">Toolbar includes the wide search bar and the YouTube header.</string>
    <string name="revanced_enable_wide_search_bar_with_header_summary_off">Toolbar includes the wide search bar without the YouTube header.</string>
```